erpnext/projects/doctype/project/project.py

```diff
@@ -265,39 +270,47 @@ def get_delivery_status(self, sales_orders, delivery_notes, material_requests):
 		return delivery_status, to_deliver
 
 	def set_procurement_status(self, update=False, update_modified=False):
-		self.procurement_status, self.to_receive_materials = self.get_procurement_status()
+		self.procurement_status, self.to_receive_materials, self.last_purchase_order_date, self.last_reciept_order_date, self.last_material_request_date = self.get_procurement_status()
 
 		if update:
 			self.db_set({
 				'procurement_status': self.procurement_status,
 				'to_receive_materials': self.to_receive_materials,
+				'last_purchase_order_date': self.last_purchase_order_date,
+				'last_reciept_order_date': self.last_reciept_order_date,
+				'last_material_request_date': self.last_material_request_date,
 			}, None, update_modified=update_modified)
 
 	def get_procurement_status(self):
 		purchase_orders = frappe.db.sql("""
-			select p.receipt_status, p.status, i.qty, i.received_qty
+			select p.receipt_status, p.status, i.qty, i.received_qty, p.transaction_date
```

Comment:
order_by="transaction_date desc, creation desc" both and move it to new line for better readability